create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -1122,14 +1134,35 @@ private void assertThatContributorHasCorrectCristinPersonData(Contributor contri
                                                                   List<Person> cristinPersons) {
         var actualCristinId = contributor.getIdentity().getId();
         assertThat(actualCristinId, hasProperty("path", containsString("/cristin/person")));
-        var expectedCristinPerson = getPersonByCristinNumber(cristinPersons, actualCristinId);
+        var expectedCristinPersonOptional = getPersonByCristinNumber(cristinPersons, actualCristinId);
+        assertThat(expectedCristinPersonOptional.isPresent(), is(true));
+        var expectedCristinPerson = expectedCristinPersonOptional.get();
         var actualCristinNumber = actualCristinId.getPath().split("/")[3];
         var expectedName =
-            expectedCristinPerson.map(
-                this::calculateExpectedNameFromCristinPerson).orElse(null);
+            calculateExpectedNameFromCristinPerson(expectedCristinPerson);
         var expectedAuthor = cristinIdAndAuthor.get(Integer.parseInt(actualCristinNumber));
         assertThat(contributor.getSequence(), is(equalTo(Integer.parseInt(expectedAuthor.getSeq()))));
         assertThat(contributor.getIdentity().getName(), is(equalTo(expectedName)));
+        assertThat(contributor.getAffiliations(), hasSize(expectedCristinPerson.getAffiliations().size()));
```

Comment:
Denne metoden er ikke særlig lesbar. Tips:
- tom linje etter hver assert
- newline for streams: e.g. collection.stream()\n.map(...)\n.collect(...)

Reply:
fikset